lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -345,16 +345,41 @@ def scoped_privileges(user_or_attribute, object_or_attribute, options = {})
       end
     end
 
+    ## Optimized version of PolicyMachine#scoped_privileges
+    # Returns all objects the user has the given operation on
+    # TODO: Support multiple policy classes here
+    def accessible_objects(user_or_attribute, operation, options = {})
+      operation = class_for_type('operation').find_by_unique_identifier!(operation.to_s) unless operation.is_a?(class_for_type('operation'))
+      permitting_oas = PolicyElement.where(id: operation.policy_element_associations.where(
```

Comment:
It feels like there should be some sneaky `or` thing or casting a record back to a relation thingamajig AR would provide that would let you sneak out of this, but I can't think of one at the moment.  Oh well.